plugins/micromega/certificate.ml

```diff
@@ -1007,6 +1007,21 @@ let gen_bench (tac, prover) can_enum prfdepth sys =
     flush o; close_out o );
   res
 
+let normalise sys =
+  List.fold_left
+    (fun acc s ->
+      match WithProof.cutting_plane s with
+      | None -> s :: acc
+      | Some s' -> s' :: acc)
+    [] sys
+
+let pre_process sys =
+  let sys1 = subst sys in
+  let sys2 = normalise sys1 in
+  let bnd = bound_monomials sys2 in
+  let sys = bnd @ saturate_by_linear_equalities sys2 @ sys2 in
+  sys
```

Comment:
The fix to #12790 might need to be applied here too.

Reply:
The rebase found it. Fixed by 1fd949ac27